text/src/autogluon/text/automm/optimization/utils.py

```diff
@@ -480,3 +492,30 @@ def apply_layerwise_lr_decay(
         parameter_group_names[group_name]["params"].append(name)
 
     return list(parameter_group_vars.values())
+
+
+def update_config_by_rules(
+    problem_type: str,
+    config: DictConfig,
+):
+    """
+    Modify configs based on the need of loss func.
+    Now it support changing the preprocessing of numerical label into Minmaxscaler while using BCEloss.
+
+    Parameters
+    ----------
+    problem_type
+        The type of the problem of the project.
+    config
+        The config of the project. It is a Dictconfig object.
+
+    Returns
+    -------
+    The modified config.
+    """
+    if problem_type == REGRESSION and OmegaConf.select(config, "optimization.loss_function") is not None:
+        loss_func = config.optimization.loss_function
+        if "bceloss" in loss_func.lower() or "bcewithlogitsloss" in loss_func.lower():
```

Comment:
Is `bceloss` a valid choice? I see only `bcewithlogitsloss` are allowed in `get_loss_func`.